packages/youtube_player_iframe/assets/player.html

```diff
@@ -44,11 +44,12 @@
 
       var platform = "<<platform>>";
       var host = "<<host>>";
+      var playerVars = "<<playerVars>>"
       var player;
       function onYouTubeIframeAPIReady() {
         player = new YT.Player("player", {
           host: host,
-          playerVars: <<playerVars>>,
+          playerVars: playerVars,
```

Comment:
I don't think the lint for HTML matters as we're using it just as an asset. Also, we want to assign object to `playerVars` instead of string.

Reply:
Alright, will revert that change. Also, on the same note, would you mind checking out my comment on this [issue](https://github.com/sarbagyastha/youtube_player_flutter/issues/589#issuecomment-1267875806)?